repository/RediStick-Core-Tests.package/RsRedisEndpointTest.class/instance/tearDown.st

```diff
@@ -1,4 +1,5 @@
 running
 tearDown
+	self sendGoodPassword.
```

Comment:
Is this *always* needed for testing basic features of redis client? 
I think authentication would be an option. 
So, there should be RsRedisClientTestCase >> testAuthenticate and RsRedisClientTestCase >> testAuthenticateFailed.

If you combine them to setup/tearDown, it is difficult to say what features are tested.

Reply:
thank for this review, I misunderstand something when I wrote this.

Reply:
I fix it in commit 
4144074

Reply:
"I think authentication would be an option."

I must authenticate the stick because I suppose that the user (dev ) will deploy a Redis server using my redis.conf file (adding a password). if I not authenticate all tests will turn to red.